waku/waku_test.go

```diff
@@ -1138,7 +1168,9 @@ func testConfirmationsHandshake(t *testing.T, expectConfirmations bool) {
 	}()
 	// so that actual read won't hang forever
 	time.AfterFunc(5*time.Second, func() {
-		rw1.Close()
+		if err := rw1.Close(); err != nil {
+			t.Errorf("error closing MsgPipe, '%s'", err)
+		}
```

Comment:
[Error handling is good](https://github.com/status-im/status-go/pull/1957/files#r417328424)